app/api.py

```diff
@@ -289,12 +289,31 @@ def get_product(product_id: int, db: Session = Depends(get_db)):
     return db_product
 
 
+@app.get(
+    "/api/v1/locations/osm/{location_osm_type}/{location_osm_id}",
```

Comment:
Do we really need the OSM type in the URL? Can't the ID uniquely identify the node?

Reply:
no sadly we need the type info, there can be nodes or ways or relations with the same ids

https://wiki.openstreetmap.org/wiki/Elements#Ids
> Users should also be aware that nodes, ways and relations share the same object id space. Any tool which is solely referring to an OSM object by its id will encounter undefined behavior (e.g. Document.getElementById()). OSM Objects must always be referred to both with their object ids and their respective object type (node, way, relation).